components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/template/BallerinaLibrary.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.apimgt.gateway.cli.model.template;
+
+/**
+ * Mustache data holder for Ballerina platform library.
+ */
+public class BallerinaLibrary {
+    private String module;
+    private String path;
+    private String artifactId;
+    private String version;
+    private String groupId;
+
+    public String getModule() {
+        return module;
+    }
+
+    public void setModule(String module) {
+        this.module = module;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+    }
+
+    public String getArtifactId() {
+        return artifactId;
+    }
+
+    public void setArtifactId(String artifactId) {
+        this.artifactId = artifactId;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    public String getGroupId() {
+        return groupId;
+    }
+
+    public void setGroupId(String groupId) {
+        this.groupId = groupId;
+    }
```

Comment:
lets remove the above gette and setters

Reply:
Why? we need those to set and get these values from `BallerinaToml`

Reply:
We don't need those values in ballerina.toml. So we can remove them, otherwise those values can give the idea that it can download the maven dependencies